docs/model-guide.md

```
@@ -80,3 +80,17 @@ By convention
 
 ### Different primary key name
 ### Different table name
+
+## Notes
+
+### JSONB columns
+JSONB columsn need to be explicitly serialized to a PGObject before being saved.  That can be done in the models with this code 
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)